### PR TITLE
added scripts for (almost) unattended crab testbed deployment

### PR DESCRIPTION
**Pull request by talamoig**
Hi,

as discussed during the dev meeting, I'm merging my scripts for semi-unattended installation (https://github.com/talamoig/crab-testbed) into this repo.

Cheers,
Ivano

### CODE REVIEW

File: scripts/crabdev-deploy/paramsrc-filecheck.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+var_check 'GISTEXTURL'
+if [ $? -ne 0 ]
+then
+    echo "If you don't have it yet, considering your current configuration, the content of the gist url should be the following:"
+    cat <<EOF
+{
+    "cmsweb-dev": {
+        "delegate-dn": [
+            "/DC=ch/DC=cern/OU=computers/CN=vocms(045|052|021|031).cern.ch|/DC=ch/DC=cern/OU=computers/CN=talamoig-taskworker.cern.ch|/DC=ch/DC=cern/OU=computers/CN=$TW_HOSTNAME"
```
Comment (juztas) [inline scripts/crabdev-deploy/paramsrc-filecheck.sh]: /DC=ch/DC=cern/OU=computers/CN=talamoig-taskworker.cern.ch this one is not needed for everyone

File: scripts/crabdev-deploy/paramsrc-filecheck.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        ],
+        "transformation": {
```
Comment (juztas) [inline scripts/crabdev-deploy/paramsrc-filecheck.sh]: transformation is also not used anymore. you can remove it

File: scripts/crabdev-deploy/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Crab Server Installation Scripts
+
+This repo contains scripts to build a CRAB 3 development testbed, assuming you're working on CERN lxplus system and interacting with the Openstack facility.
+
+The typical workflow of the first time you use this script collection is:
+  * `git clone https://github.com/talamoig/crab-testbed # in an area that will be accessible by your VMs later on, eg. on afs`
```
Comment (juztas) [inline scripts/crabdev-deploy/README.md]: Can you add also these scripts to official repo (to this pull) and change documentation?

**Comment by talamoig** [comment]
I made the requested changes and a few small ones already discussed offline.

**Comment by ericvaandering** [comment]
retest this please

**Comment by ericvaandering** [comment]
test this please

**Pull request closed without merging**